apiserver/facades/agent/credentialvalidator/register.go

```diff
@@ -24,5 +24,9 @@ func newCredentialValidatorAPI(ctx facade.ModelContext) (*CredentialValidatorAPI
 		ctx.DomainServices().Cloud(),
 		ctx.DomainServices().Credential(),
 		ctx.Resources(), ctx.Auth(),
-		ctx.Logger().Child("credentialvalidator"))
+		ctx.DomainServices().Model(),
+		ctx.DomainServices().ModelInfo(),
```

Comment:
nit: elsewhere we'd introduce a domain services var
`domainServices := ctx.DomainServices()`

Reply:
Refactored with thanks!